src/tests/ftest/slurm_setup.py

```diff
@@ -68,35 +57,36 @@ def update_config_cmdlist(args):
 
     """
     all_nodes = NodeSet("{},{}".format(str(args.control), str(args.nodes)))
+    cmd_list = [
+        "sed -i -e 's/ClusterName=cluster/ClusterName=ci_cluster/g' {}".format(
+            SLURM_CONF),
+        "sed -i -e 's/SlurmUser=slurm/SlurmUser={}/g' {}".format(
+            args.user, SLURM_CONF),
+        "sed -i -e 's/NodeName/#NodeName/g' {}".format(
+            SLURM_CONF),
+        ]
     if not args.sudo:
         sudo = ""
     else:
         sudo = "sudo"
     # Copy the slurm*example.conf files to /etc/slurm/
     if execute_cluster_cmds(all_nodes, COPY_LIST, args.sudo) > 0:
         sys.exit(1)
-    if distro in ["el8", "suse.lp153"]:
-        cmd_list = [
-            "sed -i -e 's/SlurmctldHost=linux0/SlurmctldHost={}/g' {}".format(
-                args.control, SLURM_CONF),
-            "sed -i -e 's/ClusterName=cluster/ClusterName=ci_cluster/g' {}".format(
-                SLURM_CONF),
-            "sed -i -e 's/SlurmUser=slurm/SlurmUser={}/g' {}".format(
-                args.user, SLURM_CONF),
-            "sed -i -e 's/NodeName/#NodeName/g' {}".format(
-                SLURM_CONF),
-            ]
-    else:
-        cmd_list = [
-            "sed -i -e 's/ControlMachine=linux0/ControlMachine={}/g' {}".format(
-                args.control, SLURM_CONF),
-            "sed -i -e 's/ClusterName=linux/ClusterName=ci_cluster/g' {}".format(
-                SLURM_CONF),
-            "sed -i -e 's/SlurmUser=slurm/SlurmUser={}/g' {}".format(
-                args.user, SLURM_CONF),
-            "sed -i -e 's/NodeName/#NodeName/g' {}".format(
-                SLURM_CONF),
-            ]
+    match = False
+    # grep SLURM_CONF to determine format of the the file
+    for ctl_host in ["SlurmctldHost", "ControlMachine"]:
+        if match:
+            break
```

Comment:
Minor: You could just `break` after setting `match = True`

Reply:
done